### PR TITLE
README: improve guide for custom script

### PR DESCRIPTION
**Pull request by friendly-bits**
Guide improvements as discussed in the forum.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,30 +155,33 @@ https://github.com/hagezi/dns-blocklists
 
 ## User-configurable calls on success or failure
 
-adblock-lean supports user-configurable scripts on success or failure (eg send an email/SMS/msg)
+adblock-lean supports specifying a custom script to define the functions `report_success` and `report_failure` which would be called on success or failure (can be used to eg send an email/SMS/msg)
```
Comment (lynxthecat) [inline README.md:158]:
```suggestion
adblock-lean supports specifying a custom script to define the functions `report_success` and `report_failure` to be called on success or failure (can be used to eg send an email/SMS/msg)
```

File: README.md [hunk 2]
```diff
@@ -155,30 +155,33 @@ https://github.com/hagezi/dns-blocklists
...
 
 **Example below for free Brevo (formerly sendinblue) email service, but use your favourite smtp/email/SMS etc method.**
 
 - Install mailsend package in OpenWRT
 - Sign up for free Brevo account (not affiliated!) - provides 300 free email sends per day
-- Edit your config file custom_script path.  Recommended path is '/usr/libexec/abl_custom-script.sh' which the luci app has permission to access
+- Edit your config file custom_script path.  Recommended path is '/usr/libexec/abl_custom-script.sh' which the adblock-lean luci app will have permission to access (when the luci app is ready)
```
Comment (lynxthecat) [inline README.md:164]:
```suggestion
- Edit your config file custom_script path.  Recommended path is '/usr/libexec/abl_custom-script.sh', which the adblock-lean luci app will have permission to access (for when the luci app is ready)
```

**Comment by lynxthecat** [comment]
Thanks @friendly-bits. Could you be persuaded by my small edits above?

**Comment by friendly-bits** [comment]
@lynxthecat Done.

**Pull request merged**